src/main/java/gov/usgs/cida/pubs/springinit/SpringConfig.java

```diff
@@ -73,6 +78,11 @@ public void configureMessageConverters(List<HttpMessageConverter<?>> converters)
 	public MappingJackson2HttpMessageConverter jackson2HttpMessageConverter() {
 		Jackson2ObjectMapperBuilder builder = new Jackson2ObjectMapperBuilder();
 		builder.featuresToDisable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
+		builder.featuresToEnable(DeserializationFeature.ACCEPT_EMPTY_STRING_AS_NULL_OBJECT);
+		//We are using a custom StringDeseriealizer to change empty strings into nulls.
+		//DeserializationFeature.ACCEPT_EMPTY_STRING_AS_NULL_OBJECT does not do String as
+		//empty string ("") is a valid String value. It is for other Objects.
+		builder.deserializerByType(String.class, pubsStringDeserializer);
```

Comment:
Question for my own understanding. The description mentions Postgres converting empty String to
an integer. But doesn't this change trim white space from anything string that is put in the database?
That is likely what we want, but thought I would check.

Reply:
I added the trim to remove leading/trailing whitespace. This was not called for, but I thought a reasonable expansion, just in case we ended up with only whitespace in a field.